### ISSUE TITLE
possible error in POLES ADVANCE onshore/offshore wind capacities

### ISSUE THREAD

**Issue by BTIseaver** [iiasa/ipcc_sr15_scenario_analysis#36]
Reaching out regarding something odd I noticed in the wind capacity data for POLES ADVANCE models while using the IAMC 1.5°C Scenario Explorer. For these models and scenarios, the database seems to indicate anomalously high fractions of offshore wind throughout the 21st century. A screenshot from the online Explorer highlighting this:

![image (4)](https://user-images.githubusercontent.com/70778969/106536391-310d3100-64ad-11eb-8737-10c8e603a680.png)

It looks to me like the onshore and offshore labels may have gotten transposed here? I wanted to check this, as I'm utilizing the capacities for different generation technologies for the models/scenarios in the ADVANCE project for one of my own projects.

I emailed Daniel Huppmann who took a quick look and produced some plots that seem to confirm that the onshore/offshore wind capacity values for POLES ADVANCE scenarios definitely seem odd, suggesting a reporting error:

![attachment](https://user-images.githubusercontent.com/70778969/106536384-2eaad700-64ad-11eb-9016-71448029b98e.png)

![attachment (1)](https://user-images.githubusercontent.com/70778969/106536388-2fdc0400-64ad-11eb-9cbc-c25fee725128.png)

**Comment by danielhuppmann** [comment]
Thank you @BTIseaver for spotting this issue and reporting it!

For future reference, the plots above are created with the following script (using pyam v0.10.0):
```python
import matplotlib.pyplot as plt
import pyam

df = pyam.IamDataFrame('iamc15_scenario_data_world_r2.0.xlsx')
capacity = df.filter(variable='Capacity|Electricity|Wind|*')
generation = df.filter(variable='Secondary Energy|Electricity|Wind|*')

def plot_wind(category, capacity, generation):

    fig, ax = plt.subplots(1, 2)
    _cap = capacity.filter(variable=f'*{category}')
    _cap.filter(model='POLES ADVANCE', keep=False).plot(ax=ax[0], color='blue', alpha=0.2)
    _cap.filter(model='POLES ADVANCE').plot(ax=ax[0], color='red')
    ax[0].set_title(f'Capacity {category}')

    _gen = generation.filter(variable=f'*{category}')
    _gen.filter(model='POLES ADVANCE', keep=False).plot(ax=ax[1], color='blue', alpha=0.2)
    _gen.filter(model='POLES ADVANCE').plot(ax=ax[1], color='red')
    ax[1].set_title(f'Generation {category}')

plot_wind('Onshore', capacity, generation)
plot_wind('Offshore', capacity, generation)
```

**Comment by danielhuppmann** [comment]
I reached out to the authors and will report back when I hear from them.

**Comment by kimonker** [comment]
Indeed, wind onshore and offshore have been switched in the post-processing of model outputs. I will check if this is true for all scenarios.

**Comment by danielhuppmann** [comment]
Thank you for the quick response, @kimonker - the other scenarios looked ok from my quick assessment, but I'd appreciate if you can take a more thorough look. Once I hear back from you, I'll schedule a new release where I fix these values directly via the IAMC 1.5°C Scenario Explorer database.

**Comment by kimonker** [comment]
I confirm that the issue is in the post-processing of model results and is only relevant for ADVANCE scenarios (issue also present in the original ADVANCE WP6 and the public ADVANCE synthesis scenarios databases). @danielhuppmann please correct the issue in the IAMC database and, if possible, the ADVANCE databases.
